### ISSUE TITLE
List of changes in my submission is wrong

### ISSUE THREAD

**Issue by mschnitzer** [openSUSE/open-build-service#4971]
# Issue Description
The list of changed files in my submission is wrong.

The file `passenger-5.3.0.gem` was not `Changed` but `Added` while `passenger-5.2.3.gem` was `Deleted` but doesn't appear at all.

Expected Result
============
List of changes is correct.

How to Reproduce
===============
1. Open link: https://build.opensuse.org/request/show/606555

**Comment by Ana06** [comment]
@mschnitzer are you also removing passenger-5.2.3.gem? because if that is the ase, that doesn't appear neither... :thinking:

**Comment by bgeuken** [comment]
Yes, he does. Just compare the source and target package:wink:

**Comment by mlschroe** [comment]
I don't think this is a bug, it's just "renamed and changed".

**Comment by mschnitzer** [comment]
@mlschroe I don't really get why you think this is not a bug? It provided a completely different information.

**Comment by coolo** [comment]
the idea here is to show a content diff between tars even if they were renamed to follow a version number. Compare with https://build.opensuse.org/request/show/614758

3.2.6 was removed, 3.2.7 was added - and the backend maps this to a change in 3.2.7 (compared to 3.2.6). It's one of those OBS features where you have to read the backend code to understand what the UI is showing you.

**Comment by mlschroe** [comment]
Yes, but the UI doesn't show the old name, so maybe that's what this issue is about.
(And it's not a secret backend feature, it's just how the data is displayed.)

**Comment by mschnitzer** [comment]
@coolo @mlschroe  ok, the explanation make sense to me. Not sure if it would make sense to display the old name as well. I don't see any usecase atm. What do you guys think?

**Comment by mlschroe** [comment]
I think it makes sense to display the old name if it is different from the new name.

**Comment by coolo** [comment]
IMO bug report - the UI is ommiting important infos

**Comment by mschnitzer** [comment]
@vpereira while I was thinking about it a bit more, I'd say it's a bug.
